src/lib/management/pages/StudentsPage/AddMemberDialog.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import { Dialog, DialogTitle, DialogContent, DialogActions, Button, Checkbox, FormControlLabel, List, ListItem, Typography } from '@mui/material';
+import { UserService } from 'src/services/user.service'; 
+
+const AddMemberDialog = ({ open, onClose, programRef }) => {
+  const [users, setUsers] = useState([]);
+  const [selectedUsers, setSelectedUsers] = useState([]); 
+
+  const fetchUsers = async () => {
```

Comment:
make a function in user service that called getUserByProgram and move the logic to there